build.cljs

```diff
@@ -0,0 +1,28 @@
+(ns build
+  (:require [clojure.tools.build.api :as b]
+            [deps-deploy.deps-deploy :as dd]))
+
+(def lib 'district0x/district-ui-reagent-render) ; ends up as <group-id>/<artifact-id> in pom.xml
```

Comment:
You are using the group-id district0x. Is that intentional? shouldn't be using io.github.district0x?